dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -1,58 +1,427 @@
+							headlineSize="large"
+							imageUrl={card.image}
+							imagePosition={'top'}
+							imagePositionOnMobile={'top'}
+						/>
+					</LI>
+				);
+			})}
+		</UL>
+	);
+};
+
+/**
+ * When the first 'big' is boosted & there is atleast 1 more big we switch up the layout a little -
+ *
+ * The 'standards' section (what this component renders), consists of 2 columns, rather than a wrapped row,
+ * where the first big (all other 'bigs' fallback to standards) sits in the first slot of the left hand column,
+ * and it gets a top image (left on mobile), and is followed by up to the next 2 standards.
+ * The right hand column will take (up to) the remaining 5 standards. Any more standards will not be rendered.
+ *
+ */
+const ThirdBoostedPlusBig = ({
+	standards,
+	big,
+	showAge,
+	containerPalette,
+}: {
+	standards: TrailType[];
+	big: TrailType;
+	showAge?: boolean;
+	containerPalette?: DCRContainerPalette;
+}) => {
+	// 'col1' has 1 big & up to 2 standards
+	const standardsCol1 = standards.splice(0, 2);
+	// 'col2' has up to 5 standards
+	const standardsCol2 = standards.splice(0, 5);
+
+	const manualUlStyles = css`
+		position: relative;
+		display: flex;
+		flex-direction: column;
+		flex-basis: 50%;
+		flex-grow: 1;
+		${until.tablet} {
+			flex-basis: 100%;
+		}
+	`;
+
+	/**
+	 * The row & columns in this component are a little outside of the standard of what the standard
+	 * LI and UL components we have support - it made more sense to manually create the CSS for this use case
+	 * than to expand & muddy the APIs of both components to support this layout type.
+	 */
+	return (
+		<li
+			css={css`
+				position: relative;
+				display: flex;
+				flex-basis: 100%;
+				flex-wrap: wrap;
+			`}
+		>
+			<ul css={manualUlStyles}>
 				<LI
-					padSides={true}
 					showDivider={true}
-					showTopMarginWhenStacked={true}
-					percentage="25%"
+					padSides={true}
+					padBottom={standardsCol1.length > 0}
+					padBottomOnMobile={standardsCol1.length > 0}
 				>
 					<FrontCard
-						trail={secondary}
+						trail={big}
 						containerPalette={containerPalette}
 						showAge={showAge}
-						// Overrides
-						supportingContent={secondary.supportingContent}
+						supportingContent={big.supportingContent}
+						headlineSize="medium"
+						imageUrl={big.image}
+						imagePosition="top"
+						imagePositionOnMobile="left"
 					/>
 				</LI>
-			</UL>
+				{standardsCol1.map((card, cardIndex) => {
+					return (
+						<LI
+							key={card.url}
+							showDivider={true}
+							padSides={true}
+							padBottom={cardIndex < standardsCol1.length - 1}
+							padBottomOnMobile={
+								// If there are cards in the second col, we always want padding
+								standardsCol2.length > 0 ||
+								cardIndex < standardsCol1.length - 1
+							}
+						>
+							<FrontCard
+								trail={card}
+								containerPalette={containerPalette}
+								showAge={showAge}
+								imageUrl={undefined}
```

Comment:
```suggestion
```

I think removing this is the same?

Reply:
[imageUrl is set in FrontCard](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web/components/FrontCard.tsx#L39) so if you want no image to be shown you need to explicitly set it to be undefined

Reply:
Oh, that's new. That must be part of the `FrontCard` work. Fair enough, this api does make sense, I just didn't know about it, thanks for showing it to me!